### DISCUSSION TITLE
[Projects beta] Rearranging the view tabs

### DISCUSSION THREAD

**Discussion by timidak** [community/community#7577] [Projects and Issues] [answered]
One feature I couldn't find is rearranging the view tabs. I have not found a function where you can specify the order of the tabs, nor do I have the option to drag and drop to change the order. 

I'm referring to these tabs:
<img width="619" alt="Bildschirmfoto 2021-11-11 um 10 03 51" src="https://user-images.githubusercontent.com/18395108/141269719-adf8878b-2db6-4ace-82e2-a99dc312a74f.png">

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/4803

**Comment by azenMatt** [comment, marked as answer]
Hi @timidak @Katsute  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄
